ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/emailinput/v1/emailinput/emailinput.html

```diff
@@ -0,0 +1,63 @@
+     data-sly-use.clientlib="${'/libs/granite/sightly/templates/clientlib.html'}"
+     data-sly-use.label="${renderer.labelPath}"
+     data-sly-use.shortDescription="${renderer.shortDescriptionPath}"
+     data-sly-use.longDescription="${renderer.longDescriptionPath}"
+     data-sly-use.errorMessage="${renderer.errorMessagePath}"
+     data-sly-use.questionMark="${renderer.questionMarkPath}"></sly>
+<div data-sly-use.formstructparser="com.adobe.cq.forms.core.components.models.form.FormStructureParser"
+     data-sly-use.component="com.adobe.cq.wcm.core.components.models.Component"
+     class="cmp-adaptiveform-textinput"
+     data-cmp-is="adaptiveFormTextInput"
+     data-cmp-visible="${text.visible ? 'true' : 'false'}"
+     data-cmp-enabled="${text.enabled ? 'true' : 'false'}"
+     id="${component.id}"
+     data-cmp-adaptiveformcontainer-path="${formstructparser.formContainerPath}">
+    <div data-sly-call="${label.label @componentId=component.id, labelValue=text.label.value, labelVisible=text.label.visible, bemBlock='cmp-adaptiveform-textinput'}" data-sly-unwrap></div>
+    <textarea data-sly-test.useTextarea="${text.multiLine}"
+              title="${text.tooltipVisible ? '' : text.tooltipText}"
+              aria-label="${text.label.visible && text.label.value}"
+              class="cmp-adaptiveform-textinput__widget"
+              name="${text.name}"
+              disabled="${!text.enabled}"
+              readonly="${text.readOnly}"
+              required="${text.required}"
+              placeholder="${text.placeHolder}"
+              minlength="${text.minLength}"
+              maxlength="${text.maxLength}"></textarea>
+    <input data-sly-test="${!useTextarea}"
+           title="${text.tooltipVisible ? '' : text.tooltipText}"
+           class="cmp-adaptiveform-textinput__widget"
+           aria-label="${text.label.visible && text.label.value}"
+           type="email"
+           name="${text.name}"
+           value="${text.default}"
+           disabled="${!text.enabled}"
+           readonly="${text.readOnly}"
+           required="${text.required}"
+           placeholder="${text.placeHolder}"
+           minlength="${text.minLength}"
+           maxlength="${text.maxLength}"/>
+    <div data-sly-call="${questionMark.questionMark @componentId=component.id, longDescription=text.description, bemBlock='cmp-adaptiveform-textinput'}" data-sly-unwrap></div>
+    <div data-sly-call="${shortDescription.shortDescription @componentId=component.id, shortDescriptionVisible=text.tooltipVisible, shortDescription=text.tooltip, bemBlock='cmp-adaptiveform-textinput'}" data-sly-unwrap>
+    </div>
+    <div data-sly-call="${longDescription.longDescription @componentId=component.id, longDescription=text.description, bemBlock='cmp-adaptiveform-textinput'}" data-sly-unwrap></div>
+    <div data-sly-call="${errorMessage.errorMessage @componentId=component.id, bemBlock='cmp-adaptiveform-textinput'}" data-sly-unwrap></div>
```

Comment:
do we want to keep bem of email same as that of text input?

Reply:
Had the same thoughts.. I think it should be different for flexibility - WDYT?